static/docs/get-started/connect-code-and-data.md

```diff
@@ -102,6 +102,7 @@ This is how the result should look like now:
 
 This is how `prepare.dvc` looks like internally:
 
+<!-- prettier-ignore -->
```

Comment:
remove?

Reply:
all of them below as well?

Reply:
All removed now. Also ran prettier-md on everything (notice the yaml code blocks have been formatted).